lib/counter.js

```diff
@@ -47,15 +48,16 @@ function Counter(name, help, labels) {
  * Increment counter
  * @param {object} labels - What label you want to be incremented
  * @param {Number} value - Value to increment, if omitted increment with 1
+ * @param {(Number|Date)} timestamp - Timestamp to set the counter to
  * @returns {void}
  */
-Counter.prototype.inc = function(labels, value) {
+Counter.prototype.inc = function(labels, value, timestamp) {
 	if(isNumber(labels) || !labels) {
-		return inc.call(this, null)(labels);
+		return inc.call(this, null)(labels, value);
 	}
```

Comment:
Gotcha!

Might rename the variable then so it makes more sense, I can do that later on if you don't want to do it..